x-pack/plugins/canvas/public/components/refresh_control/auto_refresh_controls.js

```diff
@@ -42,71 +43,158 @@ export class AutoRefreshControls extends Component {
         <EuiFlexGroup alignItems="center" justifyContent="spaceAround" gutterSize="xs">
           <EuiFlexItem>
             <EuiDescriptionList textStyle="reverse">
-              <EuiDescriptionListTitle>Refresh this page</EuiDescriptionListTitle>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.canvas.refresh.control.headerTitle"
+                  defaultMessage="Refresh this page"
+                />
+              </EuiDescriptionListTitle>
               <EuiDescriptionListDescription>
                 {refreshInterval > 0 ? (
                   <Fragment>
-                    <span>Every {timeDurationString(refreshInterval)}</span>
+                    <span>
+                      <FormattedMessage
+                        id="xpack.canvas.refresh.control.frequencyDescriptionTitle"
+                        defaultMessage="Every"
```

Comment:
Let's leave duration parameter here:
```
...
defaultMessage="Every {timeDuration}"
values={{ timeDuration:  timeDurationString(refreshInterval) }}
...
```

It may be a kind of context for vendors.